.github/workflows/cd.yaml

```diff
@@ -0,0 +1,35 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'feature/ghcr-package' # TEMPORARY: To make sure we merge a green PR
+    tags:
+      - 'v*'
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+
+    env:
+      DOCKER_USERNAME: mirego-builds
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Log in to registry
+        run: echo "${{ secrets.MIREGO_GITHUB_PACKAGES_ACCESS_TOKEN }}" | docker login ghcr.io -u $DOCKER_USERNAME --password-stdin
+
+      - name: Build and push image
+        run: |
+          VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+          # Strip "v" prefix from tag name
+          [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
+          # Use Docker `latest` tag convention for master
+          [ "$VERSION" == "master" ] && VERSION=latest
+          # TEMPORARY: Always build “latest”
+          VERSION=latest
+          VERSION=$VERSION make
```

Comment:
Not sure the output of Make is pertinent here? 🤔

Reply:
Right, I thought it could be useful for debugging/introspection. It produces this output:

<img width="708" alt="" src="https://user-images.githubusercontent.com/11348/154070884-bce5fa42-037c-46e2-afe7-7296a6943414.png">

But you’re right, everything shown in the `make` output is available somewhere else in the whole `make build`/`make push` output. I’ll remove it! 👍